core/controllers/admin.py

```diff
@@ -1692,6 +1693,38 @@ def put(self) -> None:
         user_services.set_username(user_id, new_username)
         user_services.log_username_change(
             self.user_id, old_username, new_username)
+
+        # Update profile picture.
+        old_fs = fs_services.GcsFileSystem(
+            feconf.ENTITY_TYPE_USER, old_username)
+        new_fs = fs_services.GcsFileSystem(
+            feconf.ENTITY_TYPE_USER, new_username)
+        if old_fs.isfile('profile_picture.png'):
+            print("****************************************")
```

Comment:
Please note that the print statements were added intentionally, please ignore them. I'm still looking for a way to test this particular part and trying to debug it further. I will update this part after debugging.
Thanks

Reply:
Just a note that I have removed the print statements but I was not able to resolve the test for this particular part. I did created a debugging doc after the discussion with @vojtechjelinek [here](https://docs.google.com/document/d/1P7_xClSPdE2Jsc--XAD-TvHxwF2W5zE-8SUA_i3dhzM/edit?usp=sharing).